websocketpp/http/impl/response.hpp

```diff
@@ -137,20 +137,19 @@ inline size_t response::consume(char const * buf, size_t len) {
 }
 
 inline size_t response::consume(std::istream & s) {
-    char buf[istream_buffer];
-    size_t bytes_read;
-    size_t bytes_processed;
     size_t total = 0;
 
     while (s.good()) {
-        s.getline(buf,istream_buffer);
-        bytes_read = static_cast<size_t>(s.gcount());
+        std::string line;
```

Comment:
I needed to rework this method to handle long responses from proxies. 
Some proxies return large HTML blobs along with the '407', so we need to sink this data before moving on with authorization.